### PR TITLE
Feature/included at

### PR DESCRIPTION
**Pull request by tkmct**
- Update IncludedAtPredicate to use Stored
- update package versions
- fix test

### CODE REVIEW

File: contracts/Predicate/plasma/lib/IncludedAt.ovm [hunk 1]
```diff
@@ -1,4 +1,8 @@
 @library
-@quantifier("proof.block${b}.range${token},RANGE,${range}")
-def IncludedAt(proof, leaf, token, range, b) :=
-  VerifyInclusion(leaf, token, range, proof, b)
+@quantifier("stored.${contract},KEY,${key}")
+def Stored(value, contract, key) := IsStored(contract, key, value)
+
+def IncludedAt(proof, leaf, token, range, b, commitmentContract) :=
```
Comment (syuhei176) [inline contracts/Predicate/plasma/lib/IncludedAt.ovm:5]: It seems we need following annotations before `IncludedAt`.

```
@library
 @quantifier("proof.block${b}.range${token},RANGE,${range}")
```

File: contracts/Predicate/plasma/CheckpointPredicate.ovm [hunk 2]
```diff
@@ -2,9 +2,9 @@ from lib import LessThan
 from lib import IncludedAt
 
 @library
-@quantifier("so.block${b}.range${token},RANGE,${range}")
+@quantifier("proof.block${b}.range${token},RANGE,${range}")
 def SU(so, token, range, b) :=
-  IncludedAt(so, token, range, b).any()
+  IncludedAt(proof, token, range, b, $commitmentContract).any()
```
Comment (syuhei176) [inline contracts/Predicate/plasma/CheckpointPredicate.ovm:7]: thanks! We should pass commitmentContract address in deploy.ts like
https://github.com/cryptoeconomicslab/ovm-contracts/blob/master/scripts/deploy.ts#L274

Comment (syuhei176) [reply contracts/Predicate/plasma/CheckpointPredicate.ovm:7]: And it not `proof` but `so`.

Comment (tkmct) [reply contracts/Predicate/plasma/CheckpointPredicate.ovm:7]: okay. I'll fix them.
Ah, I tend to forget the implicit argument passing of quantifier.

File: contracts/Predicate/plasma/lib/IncludedAt.ovm [hunk 3]
```diff
@@ -1,4 +1,6 @@
 @library
```
Comment (syuhei176) [inline contracts/Predicate/plasma/lib/IncludedAt.ovm:1]: definition of Stored was removed?

https://github.com/cryptoeconomicslab/ovm-contracts/pull/139/files#diff-280af83e88321bbe7119472397cd68fbR3

File: contracts/Predicate/plasma/CheckpointPredicate.ovm (hunk 2 above)
Comment (syuhei176) [reply contracts/Predicate/plasma/CheckpointPredicate.ovm]: thanks!

File: contracts/Predicate/plasma/lib/IncludedAt.ovm (hunk 3 above)
Comment (tkmct) [reply contracts/Predicate/plasma/lib/IncludedAt.ovm:1]: Fixed.

**Pull request merged**